pandas/tests/frame/indexing/test_where.py

```diff
@@ -159,7 +159,7 @@ def test_where_set(self, where_frame, float_string_frame):
 
         def _check_set(df, cond, check_dtypes=True):
             dfi = df.copy()
-            econd = cond.reindex_like(df).fillna(True)
+            econd = cond.reindex_like(df).fillna(False)
```

Comment:
This test was broken by no longer setting fill_value = bool(inplace) so had to update